src/main/java/org/folio/bulkops/domain/converter/HoldingsNoteListConverter.java

```diff
@@ -34,15 +46,27 @@ public List<HoldingsNote> convertToObject(String value) {
 
   @Override
   public String convertToString(List<HoldingsNote> object) {
-    return object.stream()
+    var str = object.stream()
```

Comment:
No need to introduce local variable str here - return result.

Reply:
Fixed.